runtime/doc/lua.txt

```diff
@@ -2870,6 +2870,10 @@ vim.filetype.match({args})                              *vim.filetype.match()*
         (`function?`) A function that modifies buffer state when called (for
         example, to set some filetype specific buffer variables). The function
         accepts a buffer number as its only argument.
+        (`boolean?`) Return true if a match was found by falling back to a
```

Comment:
Should it be a string in case there are other variants in the future?
```suggestion
        (`string?`) "FALLBACK" if a match was found by falling back to a
```